manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MatchTest.scala

```diff
@@ -19,347 +19,305 @@
+        }
+      }
+      section("Relationship identifier in variable length relationships") {
+        p("When the connection between two nodes is of variable length, a relationship identifier becomes an collection of relationships.")
+        query("MATCH (actor { name:'Charlie Sheen' })-[r:ACTED_IN*2]-(co_actor)\nRETURN r", assertActedInRelationshipsAreReturned) {
+          p("The query returns a collection of relationships.")
+          resultTable()
+        }
+      }
+      section("Match with properties on a variable length path") {
+        val initQuery =
+          """MATCH (charlie:Person {name:'Charlie Sheen'}), (martin:Person {name:'Martin Sheen'})
+            |CREATE (charlie)-[:X {blocked:false}]->(:Unblocked)<-[:X {blocked:false}]-(martin)
+            |CREATE (charlie)-[:X {blocked:true}]->(:Blocked)<-[:X {blocked:false}]-(martin)"""
+        p("""A variable length relationship with properties defined on in it means that all
+            |relationships in the path must have the property set to the given value. In this query,
+            |there are two paths between Charlie Sheen and his dad Martin Sheen. One of the includes a
+            |``blocked'' relationship and the other doesn't. In this case we first alter the original
+            |graph by using the following query to add ``blocked'' and ``unblocked'' relationships:""")
+        query(initQuery, assertBlockingRelationshipsAdded) {
+          p("This means that we are starting out with the following graph: ")
+          graphViz()
+        }
+        query("MATCH p = (charlie:Person)-[* {blocked:false}]-(martin:Person) " +
+          "WHERE charlie.name = 'Charlie Sheen' AND martin.name = 'Martin Sheen' " +
+          "RETURN p", assertBlockingRelationshipsAdded) {
+          initQueries(initQuery)
+          resultTable()
+          p("Returns the paths between Charlie and Martin Sheen where all relationships have the +blocked+ property set to +FALSE+.")
+        }
+      }
+    }
+    section("Shortest path", "query-shortest-path") {
+      section("Single shortest path") {
+        p("Finding a single shortest path between two nodes is as easy as using the `shortestPath` function. It's done like this:")
+        query(
+          """MATCH (martin:Person {name:"Martin Sheen"} ),
+            |      (oliver:Person {name:"Oliver Stone"}),
+            |      p = shortestPath( (martin)-[*..15]-(oliver) )
+            |RETURN p""", assertShortestPathLength) {
+          resultTable()
+          p(
+            """This means: find a single shortest path between two nodes, as long as the path is max 15 relationships long. Inside of the parentheses
+              | you define a single link of a path -- the starting node, the connecting relationship and the end node. Characteristics describing the relationship
+              | like relationship type, max hops and direction are all used when finding the shortest path. You can also mark the path as optional.""")
```

Comment:
This does not make sense to me either, and a review of the shortestPath code does not clarify it for me. I think I'll delete this line and add instead a comment about the new predicate-pull-in capability.